raisedragon-core/src/main/kotlin/com/whatever/raisedragon/domain/auth/oauth/kakao/KakaoOAuthService.kt

```diff
@@ -0,0 +1,34 @@
+package com.whatever.raisedragon.domain.auth.oauth.kakao
+
+import com.whatever.raisedragon.domain.auth.AuthService
+import org.springframework.http.HttpEntity
+import org.springframework.http.HttpHeaders
+import org.springframework.http.MediaType
+import org.springframework.stereotype.Service
+import org.springframework.web.client.RestTemplate
+
+@Service
+class KakaoOAuthService(
+    private val restTemplate: RestTemplate,
+) : AuthService {
+
+    private val KAKAO_USERINFO_REQUEST_URL = "https://kapi.kakao.com/v2/user/me"
+    override fun verifyKaKao(accessToken: String): String {
+        val headers = HttpHeaders()
+        headers.contentType = MediaType.APPLICATION_FORM_URLENCODED
+        headers.setBearerAuth(accessToken)
+
+        val request = HttpEntity(
+            null,
+            headers
+        )
+
+        val response = restTemplate.postForEntity(
+            KAKAO_USERINFO_REQUEST_URL,
+            request,
+            KakaoLoginResponse::class.java
+        )
+
+        return response.body!!.id!!
```

Comment:
```suggestion
        return response.body?.id!!
```
이렇게는 안되나요? 일단 id를 못받아오는경우 예외가 터질텐데 괜찮은가요?

Reply:
못받아올때의 에러처리를 따로 해봐야겠네요